build.gradle.kts

```diff
@@ -86,10 +86,9 @@ tasks {
         dependsOn(copyPackageJson, copyJS, copySourceMap)
         workingDir("$buildDir/node_module")
         if(Os.isFamily(Os.FAMILY_WINDOWS)) {
```

Comment:
can remove this if now

Reply:
Do you mean just Os.isFamily?

Reply:
Sorry misread it

Reply:
yeah the only reason to make it OS specific was to run 2 command in one line which youi have removed now